src/estimation/src/BerdyHelper.cpp

```diff
@@ -1673,6 +1750,22 @@ bool BerdyHelper::getBerdyMatrices(SparseMatrix<iDynTree::ColumnMajor>& D, Vecto
             m_sensorsOrdering.push_back(jointSens);
         }
 
+        if(m_options.includeCoMAccelerometerAsSensor) {
+
+            //std::cout << "Berdy helper : Considering CoM accelerometer sensor inside cacheSensorOrdering";
+            IndexRange sensorRange = this->getRangeLinkSensorVariable(COM_ACCELEROMETER_SENSOR, m_model.getLinkIndex(m_options.baseLink));
+
+            BerdySensor linkSensor;
+            linkSensor.type = COM_ACCELEROMETER_SENSOR;
+            linkSensor.id = m_options.baseLink;
+            //std::cout << "Berdy helper : CoM accelerometer sensor id " << linkSensor.id << std::endl;
+            linkSensor.range = sensorRange;
+            //std::cout << "Berdy helper : CoM accelerometer sensor size : " << linkSensor.range.size << " offset : " << linkSensor.range.offset << std::endl;
```

Comment:
Prints in this section to be removed?

Reply:
Removed in https://github.com/robotology/idyntree/pull/571/commits/f57ab3c061b91f83c5351d671132cc922ca75556